backend/src/main/java/gov/cdc/usds/simplereport/db/model/PatientLink.java

```diff
@@ -33,6 +35,14 @@ public Date getRefreshedAt() {
     return refreshedAt;
   }
 
+  public boolean isExpired() {
+    return this.getRefreshedAt().before(Date.from(Instant.now().minus(oneDay, ChronoUnit.HOURS)));
+  }
+
+  public void expire() {
```

Comment:
This is a strong indication that the data model needs a fix, yes—if #819 is the time and place for that, let's make sure to have time to discuss what the model should be before that gets super deadline-tight.